pywbem/_cim_operations.py

```diff
@@ -4909,8 +4910,11 @@ class are to be included in the returned instances, as follows:
 
           :term:`py:generator` iterating :class:`~pywbem.CIMInstance`:
           A generator object that iterates the resulting CIM instances.
-          These instances include an instance path that has its host and
-          namespace components set.
+          These instances include an instance path that always has its host and
+          namespace components. When this method uses the
+          EnumerateInstances operation, the host name is inserted into
+          the response from the WBEMConnection.host attribute because DMTF
+          EnumerateInstances does not return host name in the response.
```

Comment:
This explains how the host is always set but not why. Note that the change log announces the change would explain why.

Reply:
Made shorter but makes sense to explain this since even we missed it.

Reply:
DONE